cirq/google/engine/client/quantum.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright 2019 Google LLC
+# Copyright 2020 Google LLC
```

Comment:
Do we care about it changing the copyright years?

Reply:
I don't think it matters too much, but I think the newer year is more correct and the tool updated the year, so I will keep it that way.